client/cli/src/config.rs

```diff
@@ -206,6 +206,16 @@ pub trait CliConfiguration: Sized {
 			.unwrap_or(Ok(Default::default()))
 	}
 
+	/// Allows unsafe pruning
+	///
+	/// By default this is retrieved from `ImportParams` if it is available. Otherwise it's
+	/// false.
+	fn unsafe_pruning(&self) -> Result<bool> {
```

Comment:
Okay, so as we probably will remove this again, because this was just some temporary hack. We don't need it in the trait^^